src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -217,6 +217,10 @@
     "NAME": "Feast",
     "DESCRIPTION": "Remember stsjorbsmod:Gluttony. NL Draw !M! cards. NL Exhaust."
   },
+  "stsjorbsmod:Feint": {
+    "NAME": "Feint",
+    "DESCRIPTION": "Deal !D! Damage. NL Draw 2 Cards. NL Discard 1 Card."
```

Comment:
The terms "card" and "damage" usually aren't capitalized in card descriptions (when in doubt, follow the example of existing cards :) )